static/js/topic_page/bar_tile.tsx

```diff
@@ -42,37 +43,41 @@ interface BarTilePropType {
   statVarMetadata: StatVarMetadata;
 }
 
+interface BarChartData {
+  dataGroup: DataGroup[];
+  sources: Set<string>;
+}
+
 export function BarTile(props: BarTilePropType): JSX.Element {
   const [rawData, setRawData] = useState<GetStatSetResponse | undefined>(null);
-  const [chartData, setChartData] = useState<DataGroup[] | undefined>(null);
+  const [barChartData, setBarChartData] = useState<BarChartData | undefined>(null);
 
   useEffect(() => {
     fetchData(props, setRawData);
   }, [props]);
 
   useEffect(() => {
     if (rawData) {
-      processData(props, rawData, setChartData);
+      processData(props, rawData, setBarChartData);
     }
   }, [props, rawData]);
 
   useEffect(() => {
-    if (chartData) {
-      draw(props, chartData);
+    if (barChartData) {
+      draw(props, barChartData);
     }
-  }, [props, chartData]);
+  }, [props, barChartData]);
 
+  if (!barChartData) {
+    return null;
+  }
   return (
-    <div className="chart-container">
-      {chartData && (
-        <>
-          <div className="line-title">
-            <h4>{props.title}</h4>
-          </div>
+    <ChartTileContainer
```

Comment:
Here has no indent, whether should run the lint fix.

Reply:
thanks, just ran lint